code/modules/surgery/organs/organ_external.dm

```diff
@@ -634,9 +634,10 @@ Note that amputating the affected organ does in fact remove the infection from t
 		return
 
 	var/organ_spilled = FALSE
+	var/something_else = FALSE
```

Comment:
could you leave a comment explaining what this var is tracking for the uninformed